### ISSUE TITLE
Improve README with more text and images

### ISSUE THREAD

**Issue by Eldemarkki** [Eldemarkki/Marching-Cubes-Improved#2]

**Comment by ckempke** [comment]
I'd suggest maybe not worrying about the README specifically, but rather enabling the Wiki feature in the repository.   There's a lot more content here than is going to be expressible in a reasonably-sized README, and you may as well recruit the community to help....

**Comment by Eldemarkki** [comment]
Yes, I agree. I'm not sure if Wikis were a thing back when I created this issue, but today Wikis are definitely the better solution for documentation. The README should be used for general showcasing of the project in my opinion.
